src/main/java/seedu/address/logic/commands/UploadCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.AddCommandParser;
+import seedu.address.model.Model;
+
+/**
+ * Appends a data file containing flashcards to the current data file.
+ */
+public class UploadCommand extends Command {
+
+    public static final String COMMAND_WORD = "upload";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Uploads flashcards from a data file. "
+            + "Parameters: "
+            + "FILE PATH "
```

Comment:
Add endline to this one.
```suggestion
            + "FILE PATH \n"
```